app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+
+def copy_file(command: str) -> None:
+    new_str = command.split(" ", 2)
```

Comment:
you don't need to set separator and max split here - it will split by spaces by default . Then you don't need to set maxsplit because it may return incorrect data - it will always try to split in 3 parts
```
command = "This command is wrong"
command.split(" ", 2)
['This', 'command', 'is wrong'] # 3 elements
command.split()
['This', 'command', 'is', 'wrong'] # 4 elements